src/sentry/migrations/0486_integer_pr_comment_issue_list.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 2.2.28 on 2023-06-13 22:27
+
+import django.contrib.postgres.fields
+from django.db import migrations
+
+import sentry.db.models.fields.bounded
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+    checked = False
```

Comment:
Just a note for anyone using this as an example - Don't set `checked = False` without someone from @getsentry/owners-migrations approving